controllers/getApiLibrary.controllers.js

```diff
@@ -0,0 +1,15 @@
+const { fetchApiJSON } = require('../models/apiLibrary.models')
+
+const getApiJSON = (req, res, next) => {
+    fetchApiJSON()
+    .then((data) => {
+        res.status(200).send(data) 
```

Comment:
all data from your controllers should be sent back on a meaningful key ie. `.send({endpoints: data})`

Reply:
Resolved :)